src/Header/Nav/NavLinks/NavLinks.jsx

```diff
@@ -128,7 +128,9 @@ export function NavLinks({
 											onClick={handleNavLinkClick}
 											id={`navlink-${id}`}
 										>
-											<span aria-label={abbreviation && title}>{text}</span>
+											<span aria-label={abbreviation && title} id={id}>
```

Comment:
I think we can take the if odd here now can't we?

Reply:
ahh forgot there were two :-)